inst/python/installPythonClient.py

```diff
@@ -60,8 +61,19 @@ def main(path):
     # ...but - for some reason - pip breaks when we install the python synapse client
     # So we use 'setup' directly
     packageName = "synapseclient-1.7.2"
-    linkPrefix = "https://pypi.python.org/packages/67/30/9b1dd943be460368c1ab5babe17a9036425b97fd510451347c500966e56c/"
-    installPackage(packageName, linkPrefix, path, moduleInstallationPrefix)
+    archivePrefix=packageName
```

Comment:
add a switch controlled by an environment variable.  username and branch would also be environ var's